app/models/goal.py

```diff
@@ -4,3 +4,5 @@
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    tasks = db.relationship("Task", backref="goal", lazy=True)
```

Comment:
👍 

There are lots of interesting values that `lazy` can be set to. `True` is a synonym for `"select"`, and works great here. But check out some of the [other options](https://docs.sqlalchemy.org/en/14/orm/relationship_api.html#sqlalchemy.orm.relationship.params.lazy). 😄